playground/app.js

```diff
@@ -166,24 +166,6 @@ class MongoService {
 
 const mongoservice = new MongoService();
 const mongoBackendPlugin = servicePluginFactory(mongoservice);
-const initialPipeline = [
```

Comment:
general query for this commit: could you maybe add a small 3-liner explanation of what is done in this commit?

Reply:
GitHub does not indicate which commit you're commenting on
I assume it's about dc92f572f5fb2479dd37bb2540a3ce6d80354f84 ?
I'll add a commentary

Reply:
GitHub does not indicate which commit you're commenting on
I assume it's about 9cafa39 ?
The commit message is "PipelineSelector component in Vqb", I can add a comment, I thought the message was self-explainatory :thinking:

Reply:
See 9cd5725 :)

Reply:
Hmm, I can't find the commit. Mostly, what I meant is "describe why and how, rather than what" :)

Reply:
Here it is: https://github.com/ToucanToco/weaverbird/pull/444/commits/5932e21d277ade338801fc783b20d5002d6bd4ec
Doing modifications and fixup(squashing) has this drawback, it's difficult to follow what actually happened during the review :/